app/src/main/java/com/teamagam/gimelgimel/app/map/view/ViewerFragment.java

```diff
@@ -46,38 +41,42 @@
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         View rootView = super.onCreateView(inflater, container, savedInstanceState);
-        FragmentCesiumBinding bind = DataBindingUtil.bind(rootView);
+        com.teamagam.gimelgimel.databinding.FragmentViewerBinding bind = DataBindingUtil.bind(
```

Comment:
LOL.
My IDE just messed with me, it wouldn't accept it otherwise.
This Android Studio got some shits